tests/tracking/test_model_registry_client.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+from unittest import mock
```

Comment:
As this test split suggests, we will need to define a test split for mlflow vs mlflow-skinny